### PR TITLE
Add profile about Samiksha

### PR DESCRIPTION
**Pull request by 01samiksha**

### CODE REVIEW

File: submissions/01samiksha/aboutme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# About Me
+My name is Samiksha Gulati. I am a CSE student at UIET,Panjab University, India.
+
+# Experience
+None as such
```
Comment (divyam3897) [inline submissions/01samiksha/aboutme.md]: Full stop missing, write something.

File: submissions/01samiksha/aboutme.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+# Interests
+Programming
```
Comment (divyam3897) [inline submissions/01samiksha/aboutme.md]: Full stop.

File: submissions/01samiksha/aboutme.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+# Contact Info
+Email: [01samiksha@gmail.com](mailto:01samiksha@gmail.com)
+
+Github: [01samiksha](https://github.com/01samiksha)
+
+# Emoji
+My favorite emoji is :thinking_face:
+
```
Comment (divyam3897) [inline submissions/01samiksha/aboutme.md]: Remove empty lines.

**Review by divyam3897 — CHANGES_REQUESTED** [review]

File: submissions/01samiksha/aboutme.md (hunk 3 above)
Comment (01samiksha) [reply submissions/01samiksha/aboutme.md]: Just the last extra line, or all the empty lines in b/w

Comment (divyam3897) [reply submissions/01samiksha/aboutme.md]: you can view the .md by clicking on the view button above, so depends on you. I found all lines removed also good but last lines surely to be removed.

**Pull request closed without merging**